app/boards/shields/ergo_s1_oe/boards/nice_nano.overlay

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+ #include <dt-bindings/led/led.h>
+
+&spi1 {
+    compatible = "nordic,nrf-spim";
```

Comment:
I think this needs to be updated for pinctrl changes: https://zmk.dev/blog/2023/04/06/zephyr-3-2#move-to-pinctrl-driver. See another shield like https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/corne/boards/nice_nano.overlay as an example.

Also, you might want to duplicate this file (at least) for `nice_nano_v2.overlay` since that is commonly used these days and this file won't add support for v2.